plugins/easydns.c

```diff
@@ -79,11 +79,12 @@ static int response(http_trans_t *trans, ddns_info_t *info, ddns_alias_t *alias)
 
 	DO(http_status_valid(trans->status));
 
-	if (strstr(resp, "NOERROR") || strstr(resp, "no update required"))
+	if (strstr(resp, "NOERROR") || strstr(resp, "no update required") || strstr(resp, "OK"))
 		return 0;
-
 	if (strstr(resp, "TOOSOON"))
 		return RC_DDNS_RSP_RETRY_LATER;
+	if (strstr(resp, "TOO_FREQ"))
+		return RC_DDNS_RSP_RETRY_LATER;
```

Comment:
I think this (actually great) return code from the provider warrants a new error status.    Usually providers (DYN.com in particular) just locks your account on too frequent updates.

Could you add `RC_DDNS_RSP_TOO_FREQUENT` to error.h and error.c?  For example something like this in error.c:

   { RC_DDNS_RSP_TOO_FREQUENT,        E("DDNS warning, your update interval is set too low."      )}